sycl/doc/extensions/GroupAlgorithms/SYCL_INTEL_group_sort.asciidoc

```diff
@@ -241,77 +377,203 @@ position `i` in the ordered range.
 [source,c++]
 ----
 namespace sycl::ext::oneapi {
-  template <typename Group, typename Ptr>
-  void joint_sort(Group g, Ptr first, Ptr last); // (1)
 
-  template <typename Group, typename Ptr, typename Compare>
-  void joint_sort(Group g, Ptr first, Ptr last, Compare comp); // (2)
+  // Only available if GroupExecutor was created with a work-group and some associated scratch space
+
+  template <typename GroupExecutor, typename Ptr>
+  void joint_sort(GroupExecutor exec, Ptr first, Ptr last); // (1)
+
+  template <typename GroupExecutor, typename Ptr, typename Compare>
+  void joint_sort(GroupExecutor exec, Ptr first, Ptr last, Compare comp); // (2)
 
   template <typename Group, typename Ptr, typename Sorter>
   void joint_sort(Group g, Ptr first, Ptr last, Sorter sorter); // (3)

```

Comment:
Don't you now need a comment here, like:
```c++
// Only available if Sorter provides operator()(Group, Ptr, Ptr) overload
```

Reply:
Will it be better if I add such comment into `Constraints` sections?

Reply:
Yes, that's an even better idea.  This is actually an inconsistency in the SYCL 2020 specification that I hadn't noticed before.